apps/src/templates/instructions/TopInstructions.jsx

```diff
@@ -473,6 +475,12 @@ class TopInstructions extends Component {
   handleHelpTabClick = () => {
     this.handleTabClick(TabType.RESOURCES);
     this.recordEvent('click-help-and-tips-tab');
+    logUserLevelInteraction({
+      levelId: this.props.serverLevelId,
+      scriptId: this.props.serverScriptId,
+      interaction: 'click-help-and-tips-tab',
+      codeVersion: 'how the heck am I going to get this?!?!',
```

Comment:
- [ ] figure out how the heck I'm going to get this 🙃

Reply:
In the absence of a good way to do this, at least we have timestamps on this table and on version history (right) 🤔 which means, with some effort, we could reconstruct a timeline...

Reply:
Is it something like this: https://github.com/code-dot-org/code-dot-org/blob/bcf4c2a485f554a3590e9451a2871e9dc3a904e3/dashboard/app/controllers/ai_tutor_interactions_controller.rb#L43?